devices/boot_target.go

```diff
@@ -37,21 +27,34 @@ func bootTargetWrite(device *sdk.Device, data *sdk.WriteData) error {
 	}
 
 	if action == "target" {
-		cmd := string(raw[0])
-
-		switch strings.ToLower(cmd) {
-		case "cd":
-			logger.Debug("Set Boot Target to cd")
-		case "hd":
-			logger.Debug("Set Boot Target to hd")
-		default:
-			return fmt.Errorf("unsupported command for amt power 'state' action: %s", cmd)
+		target := string(raw[0])
+
+		supportedTargets := map[string]bool {
+			"pxe": true,
+			"hd": true,
+			"cd": true,
+		}
+
+		if supportedTargets[target] {
```

Comment:
This works and should be fine, but I don't think its idiomatic go. I think the "idiomatic" (whatever that means) way would be to define the supported targets as constants at the top of the file

```go
const (
	targetPxe = "pxe"
	targetHd = "hd"
	targetCd = "cd"
)
```

and then use a switch statement here

```go
switch target {
case targetPxe, targetHd, targetCd:
	// Run command
default:
	// Error
}
```

but, maybe see what @timfallmk has to say about whats idiomatic, because I'm just a noob.